model/dao/UserDAO/HybridUserDAO.php

```diff
@@ -470,29 +468,20 @@ public function create(UserVO $userVO) {
             return $affectedRows;
         }
         $ldapResult = ldap_get_entries($this->ldapConnect, $sr);
-        if($ldapResult["count"] == 0)
-            // user does not exist in LDAP
-            return $affectedRows;
-
-        $sql = "INSERT INTO usr (login) VALUES(" . DBPostgres::checkStringNull($userVO->getLogin()) . ")";
-
-        $res = pg_query($this->connect, $sql);
-
-        if ($res == NULL)
-            if (strpos(pg_last_error(), "unique_usr_login"))
-                throw new SQLUniqueViolationException(pg_last_error());
-            else throw new SQLQueryErrorException(pg_last_error());
+        if($ldapResult["count"] == 0) {
+            $result = new OperationResult(false);
+            $result->setResponseCode(400);
+            $result->setErrorNumber(1234);
```

Comment:
Just double checking if 1234 is correct?

Reply:
Yes, sorry, I'm not really keeping track of what the error codes are and we aren't really using them. We'd have to follow up with a general review...